$OpenBSD: patch-kio_kssl_kopenssl_h,v 1.1 2020/03/18 14:31:20 tb Exp $

Fix use of X509_get_notBefore() and X509_get_notAfter() in
kio/kssl/ksslcertificate.cpp after <openssl/x509.h> rev 1.70.

Index: kio/kssl/kopenssl.h
--- kio/kssl/kopenssl.h.orig
+++ kio/kssl/kopenssl.h
@@ -361,6 +361,16 @@ class KOpenSSLProxy { (public)
 
 
    /*
+    *   X509_getm_notBefore - start of validity
+    */
+   ASN1_TIME *X509_getm_notBefore(const X509 *a);
+
+   /*
+    *   X509_getm_notAfter - end of validity
+    */
+   ASN1_TIME *X509_getm_notAfter(const X509 *a);
+
+   /*
     *   X509_STORE_add_lookup - add a lookup file/method to an X509 store
     */
    X509_LOOKUP *X509_STORE_add_lookup(X509_STORE *v, X509_LOOKUP_METHOD *m);
